draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -338,6 +338,28 @@ and are less efficient than hash and encode methods.
 
 # Algorithm Recommendations {#recommendations}
 
+The following table lists algorithms recommended by use-case:
+
+| Application       | Requirement   | Additional Details
+|-------------------|---------------|---------|
+| SPEKE {{Jablon96}}| Naive         | H(x)*G |
```

Comment:
Is Naive supposed to be the same as injective encoding?

Reply:
@samscott89 Looks good pending this comment. Feel free to merge when ACK'd!

Reply:
Not quite. `H(x)*G` is the naive method of hashing to the curve (technically a surjective encoding), with the significant downside that the discrete logarithm of the point (with base G) is know. Which breaks many protocols but is okay for SPEKE.

Reply:
Although my assumption was that this would be the most performant, which may not even be true, in which case it doesn't need to be included at all. Need to investigate when doing an overview of the approximate runtimes.

Reply:
Oops. I failed to see the Injective Encoding application at the bottom of this table, and forgot about the entire section at the end of the document. I presumed this was a typo. Thanks for clearing it up!